Packs/AccentureCTI_Feed/README.md

```diff
@@ -0,0 +1,10 @@
+# **Accenture CTI Feed**
+ 
+This pack has 1 sub-pack/integration namely:
+- ACTI Indicator Feed
```

Comment:
no need to add these 2 lines, the pack has one integration for now.
just add a brief description of the product itself please.